lib/generators/provider/templates/.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
   - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
   - RUBY_GC_HEAP_INIT_SLOTS=600000
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+bundler_args: --no-deployment
```

Comment:
@NickLaMuro @Fryguy I still dont see the second `bundle install`. The first one is initiated by `bin/setup`, ok. But where is the "default" one? It's not in the output. 
Also, reading the docs, that @Fryguy linked

> If you want to use a different means of handling your Ruby project’s dependencies, you can override the install command.

Thats what we do. [here](https://github.com/ManageIQ/manageiq-providers-ansible_tower/blob/master/.travis.yml#L14)